app/src/main/java/org/ole/planet/myplanet/ui/exam/BaseExamFragment.java

```diff
@@ -52,7 +52,7 @@ public abstract class BaseExamFragment extends Fragment implements CameraUtils.I
     RealmSubmission sub;
     HashMap<String, String> listAns;
     boolean isMySurvey;
-    String mac_addr = NetworkUtils.getMacAddr();
+    String mac_addr = NetworkUtils.getUniqueIdentifier();
```

Comment:
Should we rename this string?

Reply:
has a number of usages in this class renaming is a good Idea